keras_core/models/model.py

```diff
@@ -448,12 +448,39 @@ def to_json(self, **kwargs):
         return json.dumps(model_config, **kwargs)
 
     def export(self, filepath, format="tf_saved_model"):
-        raise NotImplementedError(
-            "The export() method is not yet supported. It will "
-            "be added in the next version. For the time being, you "
-            "can use `tf.saved_model.save(model)` to save a "
-            "TensorFlow SavedModel for your Keras Core model."
-        )
+        """Create a SavedModel artifact for inference (e.g. via TF-Serving).
+
+        This method lets you export a model to a lightweight SavedModel artifact
```

Comment:
We should first mention that it only works with the TF backend.

Reply:
Fixed, thanks